### PR TITLE
Initial attempt at a epic5 build

### PR DESCRIPTION
**Pull request by NathanHowell**
This is enough to get it working on my box, I haven't tried to get the packaging done yet.

### CODE REVIEW

File: cross/epic5/patches/config.cache.patch [hunk 1]
```diff
@@ -0,0 +1,4 @@
+--- /dev/null     2011-08-22 20:38:12.000000000 +0200
++++ config.cache  2011-12-16 23:15:45.000000000 +0100
+@@ -0,0 +1 @@
++ac_cv_func_getpgrp_void=yes
```
Comment (Diaoul) [inline cross/epic5/patches/config.cache.patch]: Pass this as CONFIGURE_ARGS in cross

Comment (NathanHowell) [inline cross/epic5/patches/config.cache.patch]: Any idea how? I couldn't figure out a way, but it does work as an environment variable, see the most recent push.

File: cross/epic5/Makefile [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = cross/openssl cross/ncurses cross/libiconv
+
+HOMEPAGE = http://www.epicsol.org/
+COMMENT  = Internet Relay Chat client for UNIX like systems
+LICENSE  = BSD3
+
+GNU_CONFIGURE = 1
+CONFIGURE_ARGS = --with-ipv6 --with-ssl --with-wserv --with-iconv
```
Comment (Diaoul) [inline cross/epic5/Makefile]: Add `ac_cv_func_getpgrp_void=yes` to CONFIGURE_ARGS here. It should work, if not, there is something wrong elsewhere.
See other cross packages for example, there are other with configure checks overrides.

Comment (NathanHowell) [inline cross/epic5/Makefile]: It doesn't work in `CONFIGURE_ARGS`. Both other ways do work however.

Comment (NathanHowell) [inline cross/epic5/Makefile]: Alright, it does work if `configure` is regenerated. I'm pushing a fix shortly.

**Comment by Dr-Bean** [comment]
Could you please open a new PR which includes the rest of the SPK as well as this cross package? It's not very likely we'll merge this on its own.

**Comment by NathanHowell** [comment]
Sure, I won't have time to work on this again for a bit but I'll reopen it when packaging is ready.

**Pull request closed without merging**